src/commands/actions/inventory_cmd.go

```diff
@@ -62,10 +72,40 @@ func (a *inventory) Args() []string {
 		"-v", "/sys/class:/host/sys/class:ro",
 		"-v", "/run/udev:/host/run/udev:ro",
 		"-v", "/dev/disk:/host/dev/disk:ro",
+	}
 
+	// The EFI variables files system will not exist for machines that boot in BIOS mode, so we can't add it
+	// unconditionally, as that will make the podman command fail.
+	const efivarsPath = "/sys/firmware/efi/efivars"
+	efivarsLogger := logrus.WithFields(logrus.Fields{
+		"path": efivarsPath,
+	})
+	_, err := a.filesystem.Stat(efivarsPath)
+	if os.IsNotExist(err) {
+		efivarsLogger.WithFields(logrus.Fields{
+			"path": efivarsPath,
+		}).Info("EFI variables filesystem isn't mounted")
```

Comment:
```suggestion
		efivarsLogger.Info("EFI variables filesystem isn't mounted")
```
`efivarsLogger` already contains this field

Reply:
Indeed, thanks @adriengentil.

Reply:
Done.